src/js/index.js

```diff
@@ -152,7 +154,9 @@
     var nestedHeadings = nestedHeadingsObj.nest
 
     // Render.
-    buildHtml.render(options.tocSelector, nestedHeadings)
+    if (!options.skipRendering) {
+      buildHtml.render(options.tocSelector, nestedHeadings)
```

Comment:
So then do you call this somewhere else or render the toc yourself with react or something? I'd love to see the code if you don't mind sharing. I've often thought of adding react as an alternate rendering option.

Reply:
yes exactly I'm building the DOM with custom React code. It's a fully open-source project so you can see the code there: https://github.com/SocialGouv/code-du-travail-numerique/tree/feature/convention-sidebar-scroll/packages/code-du-travail-frontend/src/conventions/texte . The components that you care about are `SummaryTitle`, `SummaryItem` and the ones in the `toc` folder which are very thin wrappers for divs with classes required by tocbot.

I have to warn you that this is far from generic, there's quite a lot of business logic in there. It's displaying french law texts that have recurring structures.

Reply:
Cool, thanks for sharing!